### PR TITLE
[DISCO-2162] chore: provide shortcuts for frequently used tasks

### PR DESCRIPTION
**Pull request by tiftran**
also covers [DISCO-2156] since it's relatively small

[DISCO-2156]: https://mozilla-hub.atlassian.net/browse/DISCO-2156?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	$(POETRY) run bandit --quiet -r $(APP_DIRS)
+
+.PHONY: format
+format: install  ##  Sort imports and reformat code
+	$(POETRY) run isort $(APP_DIRS) --profile black
+	$(POETRY) run black $(APP_DIRS)
+
+.PHONY: check
+check: install
+	$(POETRY) run python manage.py makemigrations --check --dry-run --noinput
```
Comment (mtbottle) [inline Makefile:30]: name nit: can we call this `migration-check`? `check` is kind of vague and undescriptive

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,35 @@
+APP_DIRS := consvc_shepherd contile
+COV_FAIL_UNDER := 95
+INSTALL_STAMP := .install.stamp
```
Comment (ncloudioj) [inline Makefile:3]: Let's add `.install.stamp` to .git/dockerignore.

**Review by mtbottle — APPROVED** [review]
Generally LGTM! Just a small nit, but shouldn't be blocking.

**Comment by ncloudioj** [comment]
@mtbottle just made some related [changes](https://github.com/mozilla-services/merino-py/pull/166#pullrequestreview-1236526947) to merino, I think we can apply that here as well.

File: .dockerignore [hunk 3]
```diff
@@ -1,5 +1,6 @@
 __pycache__
 .env
+.install.stamp
```
Comment (ncloudioj) [inline .dockerignore:3]: Don't forget .gitignore :)

**Review by ncloudioj — APPROVED** [review]

**Pull request merged**